easybuild/easyconfigs/l/LLVM/LLVM-9.0.1-GCCcore-9.3.0.eb

```diff
@@ -0,0 +1,45 @@
+easyblock = 'CMakeMake'
+
+name = 'LLVM'
+version = '9.0.1'
```

Comment:
We should be a bit careful with the LLVM version we pick for the 2020a generation, since `numba` for example doesn't support to latest LLVM yet (see https://github.com/numba/llvmlite/issues/523)...

Reply:
well according to that issue, numba 0.31.1 should support LLVM 9, and 0.31.1 has been released.
I can start testing that a little bit.

Reply:
I can see that they stated the following: 

llvmlite versions | compatible LLVM versions
-- | --
0.29.0 - ... | 7.0.x, 7.1.x, 8.0.x

I will test llvmlite 0.31.0 and/or 0.32.0.dev against LLVM 9.0.1

Reply:
llvmlite 0.31.0 still doesn't know about LLVM 9
```
    if not (out.startswith('8.0.') or out.startswith('7.0.')
            or out.startswith('7.1.')):
```

Reply:
Same in 0.32.0dev0

Reply:
Okay, I'll downgrade it to LLVM 8.0.x

Reply:
Should I keep 9.0.1?

Reply:
I'd say we hold back numba instead. should hopefully not take too long for them to release a version that supports LLVM9

Reply:
@boegel What will you say? I've kept 9.0.1 for now but can also push 8.0.1.

Reply:
We should probably just go ahead with LLVM 9.0.1, and wait until there's a `numba` release that supports it. It's been available for a while now, we shouldn't stick to an older release because of one package not supporting the latest LLVM yet...